templates/Dockerfile.centos

```diff
@@ -2,12 +2,14 @@
 FROM centos:7
 MAINTAINER Nuxeo <packagers@nuxeo.com>
 
-# install java
-RUN yum install -y \
-       java-1.8.0-openjdk java-1.8.0-openjdk-devel 
+# install wget / tar
+RUN yum -y install wget tar
 
-# install wget
-RUN yum -y install wget 
+# install java
+RUN wget https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz
```

Comment:
- You must merge all the RUN commands
- you must verify the MD5 of the `tar.gz` (see how we do it by downloading the Nuxeo zip file)